packages/python/algod_client/tests/test_algod.py

```diff
@@ -0,0 +1,13 @@
+from algod_client import AlgodClient
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_algod_client():
+    client = AlgodClient(host="http://localhost:4001", token="a" * 64)
```

Comment:
In the end we get an intuitive way to instantiate the AlgodClient wihthout the developer needing to touch HttpClient at all